pkg/instaclustr/client.go

```diff
@@ -208,7 +208,7 @@ func (c *Client) GetRedisUser(id string) (*models.RedisUser, error) {
 	return userRedis, nil
 }
 
-func (c *Client) UpdateRedis(id string, r *models.RedisDataCentreUpdate) error {
+func (c *Client) UpdateRedis(id string, r *models.RedisUpdate) error {
```

Comment:
The message with different update fields also applies to this part of the code

Reply:
Fixed